site/src/scaffold/standard.html

```diff
@@ -8,6 +8,7 @@
 		<meta property="og:title" content="{{title}}" />
 		<meta property="og:type" content="website" />
 		<meta name="color-scheme" content="light dark" />
+		<meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-hashes' 'sha256-xcaKW0OyR0UBm0XPKdo2gWdgJW81zFPp1ovF7iW/x0c=' 'sha256-gyrAVcbmnI8a2UtMNEZq61n4Hd0+BbNaI7mhujFCsY0=' 'sha256-5lij/kYf1SCLu25NaZsAVsoxtnSyilNGmv0srOQE/NE='; connect-src 'self'; frame-src 'self' https://www.figma.com/">
```

Comment:
I was wondering if we need to add tag for `img-src` . we do have CSS targeting `img` in `markdown.css` . For now, we are not importing image in atlas but for future .  What do you think ?

Reply:
I also noticed that it looks like its blocking favicon.ico in FireFox. I think since we are changing default-src to 'self' it will set img-src to 'self' as well so we wont have to explicitly set it.